src/config.inc

```diff
@@ -76,7 +76,7 @@ endif
 OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
-CBMC_VERSION = 5.94.0
+CBMC_VERSION = 5.94.1
```

Comment:
This should be `5.95.0`.

Reply:
No, this should be a point release given that it's not a scheduled release, it's only there to include the bug fix for a build breaking bug.

Reply:
Presuming that we are planning to release based on the current develop, the release will include other user facing changes. So not just a bug fix.